overlord/ifacestate/handlers.go

```diff
@@ -31,6 +31,30 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+func (m *InterfaceManager) setupAffectedSnaps(task *state.Task, snapName string, affectedSnaps []string) error {
```

Comment:
from snapName is a bit unclear why it needs to be skipped?  affectingSnap maybe?